src/main/java/com/powsybl/openloadflow/util/ContingencyContext.java

```diff
@@ -0,0 +1,69 @@
+package com.powsybl.openloadflow.util;
+
+import com.powsybl.contingency.Contingency;
+import com.powsybl.contingency.ContingencyElement;
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Switch;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.openloadflow.sa.BranchTripping;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ContingencyContext {
```

Comment:
I don't remember why I named it `ContingencyContext`, I was not well inspired. Maybe `ContingencyPropagation` would be more meaningful?

Reply:
Or `PropagatedContingency` ?